packages/tasks_repository/tasks_data.dart

```diff
@@ -0,0 +1,30 @@
+class Task {
+final String title;
+final String description;
+final bool isDone;
+
+Task({
+required this.title,
+required this.description,
+required this.isDone,
+});
+}
+
+List tasks = [
```

Comment:
Please remember to document your code. As you remember, in the real app we would take the tasks data from the server, and this list of data is for development purposes only, so let's create a comment on this (tripple slash in FLutter) and describe that it's for dev purposes only. 
To make sure that you would not forget it, you could event create a to do in  the code like that:
`// TODO(kamil): remove it later as it's for dev purposes only`